### PR TITLE
Allow the User to Unselect a Move

### PR DESCRIPTION
**Pull request by jbartka**

### CODE REVIEW

File: services/app/src/components/gameStatus/GameInfo.jsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const GameInfo = ({ playerCount }) => {
         })}
       >
         <img src={BulletDot} className="game-info-icon" alt="bullet-dot" />
-        <img src={CheckMark} className="game-info-icon" alt="players done icon" />
+        {/* <img src={CheckMark} className="game-info-icon" alt="players done icon" /> */}
```
Comment (jvalatkevicius) [inline services/app/src/components/gameStatus/GameInfo.jsx:33]: Why is this commented?

File: services/app/src/components/player/bouncingDots/styles.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
+body {
```
Comment (jvalatkevicius) [inline services/app/src/components/player/bouncingDots/styles.css:1]: This applies style to body tag, which is global, do we really need this? If we do, it should be in some other place, not next to bouncingDots styles.

File: services/app/src/components/player/bouncingDots/styles.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+}
+
+.bouncing-loader {
+  display: flex;
+  justify-content: flex-end;
+  /* flex-grow: 1; */
```
Comment (jvalatkevicius) [inline services/app/src/components/player/bouncingDots/styles.css:11]: remove comments

File: services/app/src/components/team/TeamMember.jsx [hunk 4]
```diff
@@ -24,7 +30,8 @@ const TeamMember = ({ name, id, skipMove, color, status, removePlayer, index })
       </div>
     </div>
     {name}
-    { status === playerStatuses.FigurePlaced && <img src={CheckMark} className="team-list-item-icon" alt="player done icon" /> }
+    { (status === playerStatuses.FigurePlaced && !isAllTurnsMade) && <div className="team-list-item-icon" alt="player done icon"> <BouncingDots/></div>}
```
Comment (jvalatkevicius) [inline services/app/src/components/team/TeamMember.jsx:33]: Lets introduce a new component for player status and move all the logic related to user status display into that component.

File: services/app/src/components/player/bouncingDots/BouncingDots.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
```
Comment (jvalatkevicius) [inline services/app/src/components/player/bouncingDots/BouncingDots.jsx:1]: Lets rename the component (and the file) to `LoaderDots`.

File: services/app/src/components/gameFooter/GameFooterButtons.jsx [hunk 6]
```diff
@@ -13,9 +13,18 @@ const GameFooterButtons = ({ skipCurrentPlayerMove }) => {
...
         disabled={isAllTurnsMade || finished || !lastTurn} 
         onClick={finishMove}
+        hidden={finished}
       >
         Finish Move
       </button>
+      <button 
+        type="button" 
+        className="unselect-btn padding-x-m padding-y-s rubik-font" 
+        onClick={clearBoard}
```
Comment (jvalatkevicius) [inline services/app/src/components/gameFooter/GameFooterButtons.jsx:23]: If I understand correctly, the button clears the board for all the players, however, when I unselect my move, I would expect that only my move was reverted.

File: services/app/src/components/player/bouncingDots/BouncingDots.jsx [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+import "../bouncingDots/styles.css";
+
+const BouncingDotsLoader = (props) => {
+  return (
+    <>
+      <div className="bouncing-loader">
+        <div></div>
```
Comment (jvalatkevicius) [inline services/app/src/components/player/bouncingDots/BouncingDots.jsx:8]: You can add some classname for this div, to make it more readable, e.g. `dot`.

**Review by jvalatkevicius — CHANGES_REQUESTED** [review]

**Review by jvalatkevicius — APPROVED** [review]

**Comment by jbartka** [comment]
Done Already

**Pull request closed without merging**